### ISSUE TITLE
Get Accuracy

### ISSUE THREAD

**Issue by meltingCat** [alishdipani/NMNIST-STDP-SpiNNaker#1]
@alishdipani Hi, I would like to know how to get accuracy from the result. Will the code update in the future? Thanks.

**Comment by alishdipani** [comment]
Hi @meltingCat , thanks for opening this issue. I am not sure if I would be able to update the code in near future but I can explain how to go about implementing getting accuracy:
1. Train the network in a unsupervised manner using STDP
2. After training the network, set the learning rate to zero and simulate the network for the training set while recording last layer neuron responses for each class.
3. In the last layer, for each neuron, calculate how many number of times did that neuron spike for each class. Assign the neuron a class for which it spiked the most.
4. For testing set, simulate the network with learning rate zero. For each example, record which neuron spiked the most and then assign the class of that neuron as the class of the example.
5. Now, you have a class assigned to each example. Just calculate the accuracy as number of examples classified correctly divided by the total number of examples.

For more details refer to Section 2.6 of the paper [Diehl et al.](https://www.frontiersin.org/articles/10.3389/fncom.2015.00099/full)
